Doc/library/sqlite3.rst

```diff
@@ -518,22 +518,17 @@ Connection Objects
 
    .. method:: create_collation(name, callable)
 
-      Creates a collation with the specified *name* and *callable*. The callable will
-      be passed two string arguments. It should return -1 if the first is ordered
-      lower than the second, 0 if they are ordered equal and 1 if the first is ordered
-      higher than the second.  Note that this controls sorting (ORDER BY in SQL) so
-      your comparisons don't affect other SQL operations.
```

Comment:
I'm not sure this last sentence here is worth keeping. If the user is creating a collating function, one must assume they have more than a basic grasp of both SQL and SQLite. I'm inclined to removing it.